src/android/app/src/main/jni/native.cpp

```diff
@@ -458,6 +463,17 @@ void EmulationSession::OnEmulationStopped(Core::SystemResultStatus result) {
                               static_cast<jint>(result));
 }
 
+u64 EmulationSession::getProgramId(JNIEnv* env, jstring jprogramId) {
+    u64 program_id;
+    auto program_id_string = GetJString(env, jprogramId);
+    if (program_id_string.empty()) {
+        program_id = 0;
```

Comment:
```suggestion
        return 0;
```

that way, you can get rid of the `program_id` variable and the `else` entirely